react-basic-practice/src/components/Table/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Th,
+  Tr,
+  Thead,
+  Table as ChakraTable,
+} from "@chakra-ui/react";
+import SortingIcon from "../Icons/sorting";
+import { Column } from "src/interfaces/table";
+
+interface TableProps {
```

Comment:
naming not good. Pls add prefix `I` for interface. Like `ITableProps`

Reply:
i done